README.md

```diff
@@ -36,7 +38,9 @@ Copy the `upstart/gearman-workers.conf` and `upstart/gearman-worker.conf` files
 The workers should start and stop automatically at boot/shutdown (currently following/preceding `mysql` of startup/shutdown; may be desirable to make into configuration at some point?).
 
 To start the workers immediately:
-```# start gearman-workers```
+```bash
+start gearman-workers
```

Comment:
Just a note: The `#` bits were intentional here, to indicate a root/super-user prompt... I'm fine getting rid of 'em, just sayin'.

Reply:
yeaaaah just try'na make it consistent with formatting between prompt/script by using the `bash` notation, but then `#` makes it into a comment :/